article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -170,63 +142,27 @@ object ArticlePicker {
         "isNotPaidContent",
       ),
     )
-    article100PercentPageFeatures.forall({ case (test, isMet) => isMet })
-  }
-
-  def dcrDisabled(request: RequestHeader): Boolean = {
-    val dcrEnabled = conf.switches.Switches.DotcomRendering.isSwitchedOn
-    val forceDCROff = request.forceDCROff
-    !dcrEnabled || forceDCROff
-  }
-
-  def canConnectToDCR(): Boolean = {
-    Configuration.environment.isProd || Configuration.environment.isCode
+    article100PercentPageFeatures.forall({ case (_, isMet) => isMet })
   }
 
   def getTier(page: PageWithStoryPackage)(implicit request: RequestHeader): RenderType = {
     val primaryChecks = primaryFeatures(page, request)
-    val hasPrimaryFeatures = forall(primaryChecks)
-
-    /*
-        date: Friday 20th Nov 2020
-        id: cea453f4-9b71-435e-8a11-35ef690c7821
-        message: We are moving to exposing 90% of the audience to DCR rendering.
-        Unfortunately our experiment framework does not allow the variant group to be bigger than 50%.
-        We are then going to expose DCR to { the control group and the excluded } and reduce the size of the variant to 10%.
-     */
-    val userInDCRGroup = !ActiveExperiments.isParticipating(DotcomRendering)
-    // true if user is not participating / not in variant
+    val primaryChecksPass = primaryChecks.values.forall(identity)
```

Comment:
Is this variable the new `dcrCanRender`? From reading this cold I'm not really sure what the checks are or what 'primary' denotes

Reply:
Yeah `dcrCanRender` is a better name for this now. (`primaryChecksPass` was to align with `primaryChecks` but that doesn't really make sense anymore as these are the only checks now. Will change.